controller/controller.go

```diff
@@ -216,3 +216,11 @@ func (c *Impl) processNextWorkItem() bool {
 
 	return true
 }
+
+// GlobalResync enqueues all objects from the passed SharedInformer
+func (c *Impl) GlobalResync(si cache.SharedInformer) {
+	store := si.GetStore()
+	for _, obj := range store.List() {
```

Comment:
List() returns []interface{} - c.Enqueue expects a "namespace/name" string. You need to pass in a func to this method to get the key from the object and enqueue that instead.